src/pages/sym.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: Game Boy symbol format specification
+---
+
+import Link from '@docusaurus/Link';
+import { latestStable } from '@site/src/components/versions.js';
+
+export const LatestStableLink = ({children, to}) => <Link to={`/docs/${latestStable}/${to}`}>{children}</Link>;
+
+# Game Boy symbol (`.sym`) file format specification
+
+## Purpose & Context
+
+Game Boy ROM files contain pure binary data, but no metadata.
+In particular, symbol information is lacking when debugging or reverse-engineering such ROMs.
+Symbol files, or "sym files" for short, aim to be companions to Game Boy ROM files that provide exactly that: symbol information.
+
+Sym files have been around for a long time (the [no$gmb](http://problemkaputt.de/gmb.htm) emulator supported them, so it is safe to assume since at least the early 2000's), but the format was always loosely defined, leading to subtle incompatibilities.
+This document aims to standardize the format and usage of sym files in order to avoid this problem.
+Modifications, etc. should be suggested by [opening an issue](https://github.com/gbdev/rgbds-www/issues) or [starting a discussion](https://github.com/gbdev/rgbds-www/discussions) on this website's source repository, whichever is more appropriate.
+
+Extensions to this format are known to exist ([example](https://github.com/mattcurrie/mgbdis/#symbol-files)), but they tend to be specific to a certain use case, and are generally a backwards-compatible superset of what are described below.
+Keep in mind that some sym files are written manually, such as when disassembling ROMs, so it is recommended for consumers to be lenient in what they accept.
+
+Note also that [wla-dx's sym files](https://wla-dx.readthedocs.io/en/latest/symbols.html) are much more complex (and complete) than this format.
+RGBDS aims to keep sym files simple, and instead chooses to output other data in different files, such as map files[^1].
+
+[^1]: Map files remain thus far unspecified. Very few tools rely on information contained in them, so no specification has been written in order to keep that format extensible.
+
+## File format specification
+
+:::info
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119).
```

Comment:
I think this would be too distracting to read. The formalism isn't worth it in a doc for such a simple format, for a hobbyist tool, where implementers could probably guess most of the spec details anyway without a formal doc.

Reply:
Maybe I'm used to it? Is it really that distracting to have a capitalized word?

Reply:
I do find it so.

Reply:
Note that RFC 8174 also updates RFC 2119 to restrict its scope to IETF documents, so using the original version makes more sense, if only pedantically.